datasets/libsvm.py

```diff
@@ -20,13 +20,11 @@ class Dataset(BaseDataset):
 
     def __init__(self, dataset="bodyfat"):
         self.dataset = dataset
-        self.X, self.y = None, None
```

Comment:
I think this file should not be touched

Reply:
Actually, the change in #255 is that all the dataset direclty implement a caching mecahnism so this part is not needed anymore.